src/assets/js/react/api/fontManager.js

```diff
@@ -0,0 +1,58 @@
+import { api } from './api'
+import { serialize } from 'object-to-formdata'
+
+/**
+ * @package     Gravity PDF
+ * @copyright   Copyright (c) 2020, Blue Liquid Designs
+ * @license     http://opensource.org/licenses/gpl-2.0.php GNU Public License
+ * @since       6.0
+ */
+
+export const apiGetCustomFontList = () => {
+  const url = GFPDF.restUrl + 'fonts/'
+
+  return api(url, {
+    method: 'GET',
+    headers: {
+      'X-WP-Nonce': GFPDF.restNonce
+    }
+  })
+}
+
+export const apiAddFont = font => {
+  const url = GFPDF.restUrl + 'fonts/'
+  const formData = serialize(font)
+
+  return api(url, {
+    method: 'POST',
+    headers: {
+      'X-WP-Nonce': GFPDF.restNonce
+    },
+    body: formData
+  })
+}
+
+export const apiEditFont = ({ id, kashida, font }) => {
```

Comment:
Wouldn't it be simpler to access `font` which already contains the `id` and `kashida` properties? It would make adding new properties to edit in future much simpler too.